File: js/anagram2.js
var anagram = require("./characterMatch.js");

exports.anagramsFor = function (word, listOfWords) {
  let newList = [];
  for (let item of listOfWords) {
    if (anagram.isCharacterMatch(word, item)) {
      newList.push(item);
    }
  }
  return newList;
};

File: python/anagram2.py
from character_match import is_character_match as is_anagram

def anagrams_for(word, list_of_words):
    new_list = []
    for item in list_of_words:
        if (is_anagram(word, item)):
            new_list.append(item)
    return new_list
